extensions/amp-consent/integrating-consent.md

```diff
@@ -14,7 +14,8 @@ To add your consent management service to AMP runtime, it is expected that you:
 - Meet the restrictions that the AMP runtime applies to ensure a good user experience. These includes
   - Enforce the size of the consent prompt. The only two allowed sizes are the initial size (`width: 100vw`, `height: 30vh`), and the full screen size (`width: 100vw`, `height: 100%`) after user interactions.
   - A default placeholder will be displayed before the consent prompt iframe is ready.
-  - Enforce the size of the stored consent information. 1200 character length including storage key, consent string and additional metadata is the limit. Please [file an issue](https://github.com/ampproject/amphtml/issues/new) if you find that not sufficient.
+  - Enforce the size of the stored consent information, when the page is served from a cache. 1200 character length including storage key, consent string and additional metadata is the limit. Please [file an issue](https://github.com/ampproject/amphtml/issues/new) if you find that not sufficient.
```

Comment:
Please add a sentence to clarify that the restriction is only applied when the Storage API is used. Also include a link to the Storage API doc.